semantic-web-technology-advantages.markdown

```diff
@@ -9,28 +9,28 @@ permalink: /semantic-web-technology-advantages
+For a machine to deal with formal semantics all intended **meaning has to be stated explicitly** (no hidden assumtions). This can be challenging for a researcher, realizing that own data is not comprehensible enough and has some gaps that can only be reproduced by the creator(s) of the data model(s).
 ### Quality control
 This need for explicitness creates a **quality control feedback loop for the source data model and data**.
 ### Easier data management
-More transparant data models also lead to **easier data management**, less depending on specific persons. In this context it is worthwhile to mention that OWL-ontologies are self-describing formal vocabularies, by means of the human oriented labels and comments (definitions) of ontological elements, and documentation in queries and rules.
+More transparant data models also lead to **easier data management**, less depending on experts. In this context it is worthwhile to mention that OWL-ontologies are self-describing formal vocabularies, using human-oriented labels and comments (definitions) of ontological elements and documentation in queries and rules.
 ## Reusable consented domain knowledge
-From a pure content point of view domain knowledge becomes highly exchangeable and reusable through expression in formal domain ontologies and N3-rules.
+From a pure content point of view domain knowledge becomes highly exchangeable and reusable through the expression in formal domain ontologies and N3-rules.
 ## Machine-interpretable semantics
 The built-in logic of the W3C SW standards makes the semantics also machine-interpretable enabling automated semantic interoperability and machine reasoning.
 ### Automated semantic interoperability
-Interoperability on the semantic level in an automated way is the highest, crossing the borders of natural languages and knowledge domains.
+Interoperability on the semantic level in an automated way can cross borders of natural languages and knowledge domains.
 It has to be pointed out that for adding new knowledge in ontologies and rules human intervention is mostly still needed.
 ### Machine reasoning
-The biggest advantage, building further on the previous one, is [machine reasoning](/n3-rule-based-machine-reasoning), to make inferences of all kinds by deriving new from existing data.
+The biggest advantage, building further on the previous one, is [machine reasoning](/n3-rule-based-machine-reasoning), to make inferences of all kinds by deriving new from existing data. A very simple example is the concept of subclasses. If woman is a subclass of human and in the data it appears that "Andrea is a woman", then Machine Reasoning is used to extend that data with the entry of "Andrea is a human". And this is only the beginning. 
```

Comment:
Don't know if this is a good example but I think it would be good to give a very easy comprehensible example here to grab the reader.